Fable.Remoting.Client/Proxy.fs

```diff
@@ -134,7 +134,13 @@ module Proxy =
                     match returnTypeAsync with
                     | TypeInfo.Async getAsyncTypeArgument -> getAsyncTypeArgument()
                     | TypeInfo.Promise getPromiseTypeArgument -> getPromiseTypeArgument()
-                    | _ -> failwithf "Expected field %s to have a return type of Async<'t>" func.FieldName
+                    | TypeInfo.Any getReturnType ->
+                        let t = getReturnType()
+                        if t.FullName.StartsWith "System.Threading.Tasks.Task`1" then
+                            t.GetGenericArguments().[0] |> createTypeInfo
```

Comment:
This could go into Fable.SimpleJson, but I assumed you wouldn't want it there as tasks aren't supported in Fable.